raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
I am afraid I don't follow.

You mean this:     `assert not api0.manager_address_if_token_registered(token_addresses[0])` ? `assert not` and `assert is None` is the same thing in this case.